app/_hub/kong-inc/openid-connect/index.md

```diff
@@ -1612,7 +1612,7 @@ The OpenID Connect plugin extends the [Kong Admin API][admin] with a few endpoin
 ### Discovery Cache
 
 When configuring the plugin using `config.issuer`, the plugin will store the fetched discovery
-information to the Kong database, or in the worker memory with Db-less.
+information to the Kong database, or in the worker memory with DB-less. Discovery information remains cached until the OIDC plugin is enabled, disabled, or updated.
```

Comment:
I think this is more complicated than this. The save triggers rediscovery. But rediscovery is also controlled by `config.rediscovery_lifetime` (which limits how often `rediscovery` actually happens). Not only that. Rediscovery may also happen when plugin is presented with a token which key is not known.